pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/tenant/DefaultTenantRebalancer.java

```diff
@@ -184,16 +277,16 @@ private String createUniqueRebalanceJobIdentifier() {
     return UUID.randomUUID().toString();
   }
 
-  private Set<String> getTenantTables(String tenantName) {
+  public Set<String> getTenantTables(String tenantName) {
```

Comment:
nit: forgot to call this out last time, but can this be package private if possible and can you mark it as @VisibleForTesting if it's only exposed for testing purposes? I at least didn't see it used outside of tests

Reply:
nit: I double checked and it does look like you can remove the `public` here since the test is in the same package. that way unnecessary functions aren't public. always better to err on the side of least access